drop/modules/mae-pipeline/Snakefile

```diff
@@ -1,15 +1,12 @@
-MAE_WORKDIR = cfg.MAE.getWorkdir(str_=False)
-
-MAE_index_name = "mae-pipeline"
-MAE_index_input, MAE_index_output, MAE_graph_file, _ = createIndexRule(
-    scriptsPath=str(MAE_WORKDIR),
-    index_name=MAE_index_name
-)
+MAE_WORKDIR = cfg.MAE.renameLocalDir()
+MAE_index_input,MAE_graph_file,MAE_index_output = cfg.MAE.getModuleIndexFiles("mae-pipeline",MAE_WORKDIR)
 
 rule mae:
```

Comment:
I think this could be moved up to the `all` rule by having there if-else statements of what to include. This could ease a lot of code.